pyproject.toml

```diff
@@ -37,11 +37,10 @@ dependencies = [
 
 [project.optional-dependencies]
 "test" = [
-    "black~=23.7.0",
-    "isort~=5.12.0",
     "pre-commit~=3.3.3",
     "pytest~=7.4.0",
-    "ruff~=0.0.281",
+    # When bumping ruff, consider also bumping the ruff-pre-commit version in .pre-commit-config.yaml.
+    "ruff~=0.1.7",
```

Comment:
In the `InvokeAI` main app repo, I've included [`ruff-lsp`](https://github.com/astral-sh/ruff-lsp) as a dependency here. It's needed for editors other than VSCode to get hints in the editor. (VSCode has an extension and does not need this package).